src/web3/near.ts

```diff
@@ -79,7 +79,7 @@ export class NearConnector extends LockConnector {
                 await walletAccount.requestSignIn(
                     'evm',
                     'Balancer Exchange',
-                    undefined,
+                    'http://localhost:8080/#/nearSuccess',
```

Comment:
Maybe we should move `http://localhost:8080` into a env variable so that we don't have to modify the code when deploying to heroku. You can check how `APP_GAS_PRICE` is used if you're not familiar with this :wink:

Reply:
No clue, why I didn't do this in the first place, fixed in amend, tried to test it out (and seemed to work), but my browser is still crashing, so I'd appreciate it if you can give it a go to make sure.